packages/core/card.ts

```diff
@@ -508,147 +561,69 @@ export class Card {
 
   private async applyRulesForAttribute(
     attributes: AttributesObject,
-    field: string,
-    rules: OcclusionRulesOrDefaults,
+    field: FieldCard,
+    rules: OcclusionRules,
     data: any,
-    included: Map<string, OcclusionRulesOrDefaults[]>
+    included: Map<string, OcclusionRules[]>
   ): Promise<void> {
-    let includeFields = await this.includeFieldsForField(rules, field);
-    if (includeFields === 'everything' || includeFields === 'upstream') {
-      attributes[field] = cloneDeep(data);
-    } else if (rules) {
-      if (!includeFields.length) {
-        return;
-      }
-      if (includeFields.find(i => i.name === field) && !isObjectLike(data)) {
```

Comment:
`isObjectLike` here was a problem. It's not a reliable way to test for "scalar" or "basic" fields, because cards are free to represent those as POJOs too. The reliable test is `this.hasFeature('field-deserialize')`